rules/integrations/kubernetes/execution_user_exec_to_pod.toml

```diff
@@ -43,8 +43,7 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset:"kubernetes.audit_logs" 
-  and kubernetes.audit.objectRef.resource:"pods" 
+kubernetes.audit.objectRef.resource:"pods" 
```

Comment:
This is a side note. We may need to revisit these rules at a later date when `event.dataset:"kubernetes.audit_logs"` is available because the `related_integrations` field leverages this to identify what package and integration is needed for this to work in Kibana. The query as is seems fine though.

Reference: https://github.com/elastic/detection-rules/pull/2060